plugins/catalog/src/components/CatalogTable/CatalogTable.test.tsx

```diff
@@ -66,11 +66,11 @@ describe('CatalogTable component', () => {
         />,
       ),
     );
-    expect(
-      await rendered.findByText(`Owned (${entites.length})`),
-    ).toBeInTheDocument();
-    expect(await rendered.findByText('component1')).toBeInTheDocument();
-    expect(await rendered.findByText('component2')).toBeInTheDocument();
-    expect(await rendered.findByText('component3')).toBeInTheDocument();
+    waitFor(() => {
```

Comment:
`waitFor` returns a promise. So we probably want to `await` it to make sure it resolves. I'm not sure this would actually fail?

Reply:
Good eye - maybe the test still isn't fixed then...

Reply:
fell back to regular get instead!